frontend/cypress/integration/common/istio_config.ts

```diff
@@ -18,6 +18,25 @@ function labelsStringToJson(labelsString: string) {
   return `{${labelsJson}}`;
 }
 
+// I included this, because the URL parameter is in plural, but the the Type itself in Kiali is singular
+// This works for all of the types currently present in Kiali (Feb 8 2023), but may break in the future, because
+// it does not support all of the english words  
+function singularize(word:string) {
+  const endings = {
+      ves: 'fe',
+      ies: 'y',
+      i: 'us',
+      zes: 'ze',
+      ses: 's',
+      es: 'e',
+      s: ''
+  };
+  return word.replace(
+      new RegExp(`(${Object.keys(endings).join('|')})$`), 
+      r => endings[r]
+  );
+}
```

Comment:
Hopefully I refactored everything correctly. I am not sure about this function though due to reasons mentioned in the code. On the other hand, I feel like installing a JS library or having tens of lines just to support this one test would be too much.